src/pretix/plugins/banktransfer/tasks.py

```diff
@@ -66,31 +66,16 @@ def cancel_old_payments(order):
 @transaction.atomic
 def _handle_transaction(trans: BankTransaction, code: str, event: Event=None, organizer: Organizer=None,
```

Comment:
``event`` no longer is optional, so we should drop the ``=None``, organizer is no longer used so we should drop it